packages/client/src/__tests__/integration/happy/chmod/test.ts

```diff
@@ -1,13 +1,17 @@
 import { getPlatform } from '@prisma/get-platform'
 import fs from 'fs'
 import path from 'path'
+import {
+  ClientEngineType,
+  getClientEngineType,
+} from '../../../../runtime/utils/getClientEngineType'
 import { generateTestClient } from '../../../../utils/getTestClient'
 
 // Tests that no error is being thrown when the binary is manually set to chmod 644 because Client fixes that itself
 test('chmod', async () => {
   await generateTestClient()
   const platform = await getPlatform()
-  if (!process.env.PRISMA_FORCE_NAPI) {
+  if (!(getClientEngineType() === ClientEngineType.NodeAPI)) {
```

Comment:
A bit weird written like this I think, why not like this?
```suggestion
  if (getClientEngineType() !== ClientEngineType.NodeAPI) {
```